UI/win-update/updater/CMakeLists.txt

```diff
@@ -22,13 +22,14 @@ target_sources(
           updater.manifest
           updater.rc)
 
-target_compile_options(updater PRIVATE $<IF:$<CONFIG:DEBUG>,/MTd,/MT> "/utf-8")
```

Comment:
The `DEBUG` config portion was just required to fix `zstd`s hardcoded runtime library mismatch, for which a linker option is used instead.

Reply:
Do you mean the `/IGNORE:4098` below? And did I understand the `"/utf-8"` part correctly?

Reply:
Correct, `/utf-8` is enabled globally, the runtime library switch is unnecessary because of the silencing of `4098`.

But I'm starting to gravitate towards replacing that with `/NODEFAULTLIB:<library>` instead to be specific about which "wrong" default library definition to ignore.

Reply:
I briefly looked into `/NODEFAULTLIB:<library>` and saw some cautions, but didn't try implementing it. We'll see how this goes.